src/xmpp/message.c

```diff
@@ -598,10 +598,6 @@ message_send_chat_otr(const char* const barejid, const char* const msg, gboolean
         stanza_attach_state(ctx, message, state);
     }
 
-    stanza_attach_carbons_private(ctx, message);
```

Comment:
Please move the OTR change in a separate commit. So that you have one commit per change that belongs together.

Reply:
Done.

Reply:
Ah yes :) But now we have 1) the not optimal commit 2) the revert commit 3) one good commit 4) another good commit. And the last 2 we actually don't need in the history. You can drop the first two commits or ammend them. Using something like `git rease -i HEAD~5`. Maybe google the details. It's quite handy to know about it. Afterwards you will have to do a `git push -f` to force push it on this branch.